### PR TITLE
Lokisk1155/home page css+stories skeleton

### PR DESCRIPTION
**Pull request by lokisk1155**

### CODE REVIEW

**Review by dmallon1 — COMMENTED** [review]
Texted you this as well, but for this pr if you really want it to just be a light skeleton I would not make any changes in `app`, `config`, or `db`

File: frontend/src/store/story.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+};
+
+export const storiesReducer = (previousState = {}, action) => {
+  switch (action.type) {
+    case ADD_STORY:
+      return { ...previousState, [action.payload.id]: action.payload };
+    default:
+      return previousState;
+  }
+};
```
Comment (dmallon1) [inline frontend/src/store/story.js:25]: it'd be great to add some unit tests for these reducers

Comment (dmallon1) [reply frontend/src/store/story.js:25]: probably doesn't make sense in this pr tho

Comment (lokisk1155) [reply frontend/src/store/story.js:25]: I will look into redux testing

**Pull request merged**